community/cypher/cypher/src/test/java/org/neo4j/cypher/internal/javacompat/NotificationAcceptanceTest.java

```diff
@@ -126,19 +131,15 @@ public void shouldNotifyWhenUsingCreateUniqueWhenCypherVersionIs3_4()
         result.close();
     }
 
-    //TODO unignore when supporting 3.3
-    @Ignore
-    public void shouldNotifyWhenUsingCreateUniqueWhenCypherVersionIs3_3()
+    @Test
+    public void shouldGetErrorWhenUsingCreateUniqueWhenCypherVersionIs3_3()
     {
+        // expect exception
+        thrown.expect( QueryExecutionException.class );
+        thrown.expectMessage( "CREATE UNIQUE is no longer supported. You can achieve the same result using MERGE");
+
         // when
         Result result = db().execute( "CYPHER 3.3 MATCH (b) WITH b LIMIT 1 CREATE UNIQUE (b)-[:REL]->()" );
```

Comment:
You are not using `result` anymore so you can call `db().execute( "CYPHER 3.3 MATCH (b) WITH b LIMIT 1 CREATE UNIQUE (b)-[:REL]->()" );` without storing the result in a variable